docs/permissions.md

```diff
@@ -54,7 +54,6 @@ private fun FeatureThatRequiresCameraPermission() {
                     "Camera permission required for this feature to be available. " +
                         "Please grant the permission"
                 }
-
                 Text(textToShow)
                 Button(onClick = { state.launchPermissionRequest() }) {
```

Comment:
```suggestion
                Button(onClick = { cameraPermissionState.launchPermissionRequest() }) {
```

Reply:
And another mention in L46. Could you change those two? Thank you

Reply:
Sure thing, thanks for the suggestion!